components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -350,6 +350,8 @@ interface BraveWalletService {
   AddUserAsset(ERCToken token, string chain_id) => (bool success);
   RemoveUserAsset(string contract_address, string chain_id) => (bool success);
   SetUserAssetVisible(string contract_address, string chain_id, bool visible) => (bool success);
+  IsBraveCryptoWalletInstalled() => (bool installed);
```

Comment:
`IsCryptoWalletsExtensionInstalled`

Reply:
fixed in 615ec530a09034306bb5efdea47492e0464e5558